terraform/operations/versions.tf

```
@@ -0,0 +1,14 @@
+# tf versions
+terraform {
+  required_version = "~> 1.7.5"
```

I found that this needed an update based on the latest Terraform from https://developer.hashicorp.com/terraform/tutorials/aws-get-started/install-cli
```suggestion
  required_version = "~> 1.10.0"
```